### PR TITLE
Dj branch

### PR DESCRIPTION
**Pull request by samson-olofinsae**
Pls help inspect the comment. thanks.

### CODE REVIEW

File: testing_template.html [hunk 1]
```diff
@@ -27,7 +27,18 @@ <h1>Hello</h1>
 
 <p> In view.py you can see what the greeting variable looks like.</p> -->
 
+<!-- 
 
+template for IF/ELSE TAG
+{% if greeting ==1%}
```
Comment (Olofinsae) [inline testing_template.html:33]: shouldn't this line following be indented?

File: testing_view.py [hunk 2]
```diff
@@ -33,3 +33,23 @@ def testing(request):
...
+def testing (request):
+    template = loader.get_template ('template.html')
+    context = {
+    'greeting' : 1,
+}
+
+    return HttpResponse (template.render (context, request))
+
+# view for IF/ELSE  tag
+#def testing (request):
```
Comment (Olofinsae) [inline testing_view.py:48]: how about identation?

**Review by Olofinsae — CHANGES_REQUESTED** [review]
pls inspect indentation. what code editor are you using -nano or vscode?  thanks.

**Review by Olofinsae — CHANGES_REQUESTED** [review]
identation

**Review by Olofinsae — APPROVED** [review]
indentation added as suggested . Good to go!

File: testing_template.html (hunk 1 above)
Comment (samson-olofinsae) [reply testing_template.html:33]: indentation added

File: testing_view.py (hunk 2 above)
Comment (samson-olofinsae) [reply testing_view.py:48]: indentation not neccesary  for commented-out session

**Review by Olofinsae — APPROVED** [review]
Code can be merged. thanks.

**Comment by Olofinsae** [comment]
Code can be merged. Good work @samson-olofinsae

**Review by Olofinsae — APPROVED** [review]
ok

**Review by Olofinsae — APPROVED** [review]

**Comment by samson-olofinsae** [comment]
Thanks @Olofinsae for approving the change.

**Pull request merged**